index.html

```diff
@@ -34,6 +34,80 @@
+          <time datetime="2020-2-6">2 mei 2020</time>
+        </p>
+      </div>
+
+      <div class="flex w-1/2 items-center">
+        <div class="lg:flex-grow text-right">
+          <a href="#responsive-header"
+            class="block lg:inline-block lg:mt-0 text-black hover:text-gray-600 hover:border-b-2 border-blue-600 mr-4 uppercase">
+            digitale krant
+          </a>
+          <a href="#responsive-header"
+            class="block lg:inline-block lg:mt-0 text-black hover:text-gray-600 hover:border-b-2 border-blue-600 mr-4 uppercase">
+            services
+          </a>
+          <a href="#responsive-header"
+            class="block lg:inline-block lg:mt-0 text-black hover:text-gray-600 hover:border-b-2 border-blue-600 mr-4 uppercase">
+            inloggen
+          </a>
+          <a href="#responsive-header"
+            class="block border border-blue-600 bg-blue-600 p-1 pr-2 pl-2 rounded lg:inline-block lg:mt-0 text-white hover:bg-blue-400 hover:border-blue-400 mr-4 uppercase">
+            abonneren
+          </a>
+        </div>
+      </div>
+    </div>
+
+    <div class="flex">
+      <div class="flex w-5/6 border-b">
+        <nav>
+          <ul class="flex">
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase"
+                href="#">voorpagina</a>
+            </li>
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase"
+                href="#">coronavirus</a>
+            </li>
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase" href="#">sport</a>
+            </li>
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase" href="#">geloof</a>
+            </li>
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase"
+                href="#">samenleving</a>
+            </li>
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase" href="#">economie</a>
+            </li>
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase" href="#">opinie</a>
+            </li>
+            <li class="mr-6">
+              <a class="text-black hover:text-gray-600 hover:border-b-2 border-blue-600 uppercase" href="#">cultuur</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="flex w-1/6 text-right text-black border rounded border-t border-r border-b border-l mr-4">
+        <input type="text" placeholder="Zoeken..">
+      </div>
+
+    </div>
+  </div>
+
   <div class="container p-5 mb-48 mx-auto">
```

Comment:
Je mag wel even kijken hoe ik het aangepakt heb met de `grid`. Zoals je ziet heb ik op de meeste plekken een padding of margin gebruikt van `5`.